playwright-e2e/pages/pancan/enrollment/consent-form-page.ts

```diff
@@ -96,12 +102,12 @@ export default class ConsentFormPage extends PancanPage {
    * @returns {Promise<void>}
    */
   async fillInParentData() {
-    const parentFirstName = new Input(this.page, { ddpTestID: PatientsData.child.ddpTestID.assentFirstName });
-    const parentLastName = new Input(this.page, { ddpTestID: PatientsData.child.ddpTestID.assentLastName });
-    const parentSignature = new Input(this.page, { ddpTestID: PatientsData.child.ddpTestID.assentSignature });
-    const relationshipChild = new Question(this.page, { prompt: PatientsData.child.ddpTestID.relationshipChild });
-    const firstName = generateUserName(user.patient.firstName);
-    const lastName = generateUserName(user.patient.lastName);
+    const parentFirstName = new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.assentFirstName });
+    const parentLastName = new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.assentLastName });
+    const parentSignature = new Input(this.page, { ddpTestID: PatientsData[this.typePatient].ddpTestID.assentSignature });
+    const relationshipChild = new Question(this.page, { prompt: PatientsData[this.typePatient].ddpTestID.relationshipChild });
+    const firstName = generateUserName(user[this.typePatient].firstName);
+    const lastName = generateUserName(user[this.typePatient].lastName);
```

Comment:
Fixed bug: `child` was hard-coded so it was not re-useable for adult enrollment.